source/includes/fact-natural-language-querying-maxTimems.txt

```diff
@@ -0,0 +1,5 @@
+When using natural language querying, you may need to modify the 
+query's maximum execution time by adjusting the :guilabel:`MAX TIME MS` 
```

Comment:
[question]

Are we suggesting that users raise or lower the max time ms value?

Reply:
This is for both cases so I intentionally left this a bit vague.
- You may need to increase the value if the AI returns a complicated multi stage pipeline that runs on a large collection and the query time exceeds the default 60 seconds.
- At the same time you may want to have a safeguard for the maxtime a query can run if the collection is not indexed correctly or is in a highly resource constrained environment.

Do you think it would be worth adding this detail to the include file?

Reply:
I think if we include a bit more information as to why users need to adjust the Max Time MS value, that would be helpful. What if we included a sentence along the lines of

> It can be difficult to predict the complexity of the query or pipeline returned by the Compass AI. You may need to adjust the MAX TIME MS query bar option to either allow longer query processing time or prevent long-running queries, depending on your application's needs.

Reply:
Will do thanks Jeff